nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/main/java/org/nuxeo/ecm/automation/core/operations/users/CreateOrUpdateGroup.java

```diff
@@ -120,6 +133,9 @@ public void run() throws OperationException {
                 throw new OperationException("Cannot create already-existing group: " + groupName);
             }
             create = false;
+
+            // make sure the group can be updated
```

Comment:
i think the comment is not required as the name of the method is explicite, or if we need to keep it perhaps add `make sure the group can be created or updated`

Reply:
Here we are just checking that we can update the group, as we are in the `if` about updating.